server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/EmailHeader.scala

```diff
@@ -22,7 +22,10 @@ package org.apache.james.jmap.mail
 import org.apache.james.mime4j.stream.Field
 
 object EmailHeader {
-  def apply(field: Field): EmailHeader = EmailHeader(field.getName, field.getBody)
+  def apply(field: Field): EmailHeader = EmailHeader(EmailHeaderName(field.getName), EmailHeaderValue(field.getBody))
 }
 
-case class EmailHeader(name: String, value: String)
+case class EmailHeaderName(value: String)
```

Comment:
To avoid boxing, it should be a value class i.e extends AnyVal

Reply:
Nice I always forget about it!